src/base.css

```diff
@@ -1357,6 +1359,22 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Styles to support the mention of Unicode characters and character sequences *****/
+   /* For more information, see: https://www.w3.org/TR/international-specs#char_ref  */
+
+	.codepoint bdi {
+		line-height: 1em;
+		font-size: 125%;
+		padding-inline: 0.25em;
+	}
+
+	.uname {
+		font-variant: all-small-caps;
+		font-size: 100%;                    
+		letter-spacing: 0.03em;
+		color: brown;
+	}
+
 /** Outdated warning **********************************************************/
 
 .outdated-spec {
```

Comment:
Give me a short while to update the editor guidelines, so that you can use the examples here.